src/core/api/io_services/v1/_swagger.json.tpl

```diff
@@ -1044,6 +1094,36 @@
         "auth_code"
       ]
     },
+    "ThirdPartyData": {
+      "type": "object",
+      "description": "Payload containing all information needed to retrieve and visualize third party message details",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Unique id for retrieving third party enriched information about the message",
+          "minLength": 1
+        },
+        "original_sender": {
+          "type": "string",
+          "description": "Either a ServiceId or a simple string representing the sender name",
+          "minLength": 1
+        },
+        "original_receipt_date": {
+          "$ref": "#/definitions/Timestamp"
+        },
+        "has_attachments": {
```

Comment:
there will also be `has_precondition`

Reply:
[38949ab](https://github.com/pagopa/io-infra/pull/684/commits/38949abb1fc690ebc0b13f39407dbcce86da0086)